Scripts/freight_lem.py

```diff
@@ -16,14 +16,11 @@
 from datahandling.matrixdata import MatrixData
 
 from datahandling.traversaldata import transform_traversal_data
-from utils.freight_costs import calc_rail_cost, calc_road_cost, calc_ship_cost
 from parameters.commodity import commodity_conversion
 
-BASE_ZONEDATA_FILE = "freight_zonedata.gpkg"
-
 
 def main(args):
-    base_zonedata_path = Path(args.baseline_data_path, BASE_ZONEDATA_FILE)
+    base_zonedata_path = Path(args.baseline_data_path, args.forecast_data_path)
```

Comment:
```suggestion
    base_zonedata_path = Path(args.forecast_data_path)
```

I do not think we need the baseline_data_path here. Could we remove that argument all together?

Reply:
Good catch, you're correct that we have no need for this argument as there are not baseline datas in freight lem runs.